workshops/meme-generator/README.md

```diff
@@ -0,0 +1,385 @@
+---
+name: 'Meme Generator'
+description: 'Spin up your own Meme Generator!'
+author: '@aaryanporwal'
+img: 'https://cloud-qmqu378mz.vercel.app/0image.png'
+---
+
+# Meme Generator
+
+Ever wondered how to make memes? Or did you ever wanted to make your _own_ cool memes and share it with the world?
+
+Then you're at the right place, today we'll be spinning up our own ***Meme Generator***, or as I like to call it - 'The Meme-Genie'
+
+Let's take a look of how our Meme Generator looks like:
+
+![Meme Generator demo](https://cloud-i6md9xxdx.vercel.app/0image.png)
+
+Source code can be found [here](https://repl.it/@aaryanporwal/Meme-generator).
+
+## Part 1: Prerequisites
+
+Basic knowledge of:
+
+1. HTML
+2. CSS
+3. JavaScript
+
+## Part 2: Setting Up The Environment
+
+We'll be using an online code editor called [repl.it](https://repl.it) for this workshop.
+
+To get started, go to [https://repl.it/languages/html](https://repl.it/languages/html). Your coding environment will spin up in just a few seconds!
+
+Now let's move on to the next part!
+
+![Excited doge](https://cloud-4560ajsyp.vercel.app/0giphy.gif)
+
+## Part 3: Building The Project
+
+### 1) HTML
+
+Let's first start by making a structure for our Meme Generator website.
+
+We'll be writing all our below code inside our `<body>` tag:
+
+* First give an appropriate heading to our website using the `<h1>` tag.
+
+* Next, want our memes to have two text fields: ***Top Text*** and ***Bottom Text***, something like this:
+
+    ![Meme showing top text and bottom text](https://cloud-ejplvs3dg.vercel.app/0image.png)
+
+    For that we'll require two text fields in our website, to achieve that we'll use `<textarea>` tag:
+
+    ```html
+    <textarea id="top-text"></textarea>
+    <textarea id="bottom-text"></textarea>
+    ```
+    
+    (put this after our `<h1>` tag)
+
+* Next, we'll need to take a meme template as a file input, we'll use:
+  
+  ```html
+  <input type="file" id="image-input" accept="image/*">
+  ```
+
+    This will pop-up a button on our website that'll let the user input a meme template as an image.
+
+* Now we need a generate button:
+
+    ```html
+    <button id="generate-btn">Generate!</button>
+    ```
+
+    Later we can add an event listener to this button which can run a function on click!
```

Comment:
This isn't directly relevant to what you're doing in this part of the workshop, so it may confuse people